### ISSUE TITLE
General questions

### ISSUE THREAD

**Issue by orisaacs** [tensorlayer/hyperpose#285]
Hi, thanks for the hard work on the package, I have a few questions about the training:
1. Are the default parameters the ones used for the example models attached (specifically the .uff model)? Also what is the default model backbone?
2. Is it possible to train a model on the COCO + Foot database? It seems like both the training and the attached models work for COCO only.

**Comment by Gyx-One** [comment]
sorry I just noticed this issue!
1.yes, the default parameters  including learning rate, batch size etc are going to produce the example models attached(both onnx and uff models).
To determine a hyerpose model, one should choose model type(using *Model.set_model_type* function) and choose model network backbone(using *Model.set_model_backbone* function),  each model type will contains a default model backbone:
for **Openpose**  the default backbone is *Vgg19*
for **LightweightOpenpose**  the default backbone is *Dailated Mobilenet*
for **PoseProposal** the default backbone is "Resnet18"
for **MobilenetThinOpenpose** the defualt backbone *MobilenetV2 variant*.
if you want to reimplement the uff model, simply set model type with **Config.MODEL.LightweightOpenpose** with model backbone "Config.BACKBONE.Vggtiny" will work!

 2.sorry, we don't support conbine dataset now, but currently we are working on it!
for now, we support train and evaluate on COCO and MPII dataset, but in July, we will add more datasets and try to support
combination of several dataset. wait for our good news!

Thanks for using our library, if you have any question, don't hesitate to contact me!

**Comment by korejan** [comment]
> 
> **2.sorry, we don't support combining dataset now, but currently we are working on it!**
> for now, we support train and evaluate on COCO and MPII dataset, but in July, we will add more datasets and try to support
> combination of several dataset. wait for our good news!
> 

Hi, what's the current situation on this? the project I'm working needs foot poses eventually, it's the most important part. In the C++ API, hyperpose::human_t is parameterized for different keypoint sizes, is there a pretrained model that does have foot poses or anyway to take a pretrained model and agument it with foot database, or retain with it?

I would like to have the hao28-600000-256x384 model with foot poses, I can't seem to find any reference to where this came from, where the training data is.

**Comment by ganler** [comment]
@korejan Hi, we haven't planed to support other forms of poses yet. As far as I am concerned, the openpose project seemed to support this.

We are still trying to enhance the human pose part(performance, accuracy, custom dataset, pip package, other backends(as TensorRT is not deployment-friendly), and Python binding of C++ inference) And we honestly hope that we can have more contributors to work on this together.

**Comment by salvador-blanco** [comment]
I have a general question, what is the difference between the model type LightweightOpenpose and OpenPose. Is there a difference in the algorithm or implementation?

**Comment by ganler** [comment]
@salvador-blanco https://arxiv.org/abs/1811.12004 This is the paper of lightweight openpose.
